cmd/security-secrets-setup/edgex-stack-security.yml

```diff
@@ -0,0 +1,583 @@
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+
+  config-seed:
+    image: edgexfoundry/docker-core-config-seed-go:1.0.0
+    container_name: edgex-config-seed
+    hostname: edgex-core-config-seed
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-core-config-seed
+    volumes:
+      - db-data:/data/db
+      - log-data:/edgex/logs
+      - consul-config:/consul/config
+      - consul-data:/consul/data
+    depends_on:
+      - volume
+      - consul
+
+  vault:
+    image: vault:1.2.3
+    container_name: edgex-vault
+    hostname: edgex-vault
+    networks:
+      edgex-network:
+        aliases:
+            - edgex-vault
+    ports:
+      - "8200:8200"
+    cap_add:
+      - "IPC_LOCK"
+    command: >
+      /usr/bin/dumb-init -- /bin/sh -c 
+      "while test \! -f /run/edgex/secrets/edgex-vault/.security-secrets-setup.complete; do
+        sleep 1;
+       done;
+       exec /usr/local/bin/docker-entrypoint.sh server -log-level=info"
+    environment:
+      - VAULT_ADDR=https://edgex-vault:8200
+      - |
+        VAULT_LOCAL_CONFIG=
+          listener "tcp" { 
+              address = "edgex-vault:8200" 
+              tls_disable = "0" 
+              cluster_address = "edgex-vault:8201" 
+              tls_min_version = "tls12" 
+              tls_client_ca_file ="/run/edgex/secrets/edgex-vault/ca.pem" 
+              tls_cert_file ="/run/edgex/secrets/edgex-vault/server.crt" 
+              tls_key_file = "/run/edgex/secrets/edgex-vault/server.key" 
+              tls_perfer_server_cipher_suites = true 
+          } 
+          backend "consul" { 
+              path = "vault/" 
+              address = "edgex-core-consul:8500" 
+              scheme = "http" 
+              redirect_addr = "https://edgex-vault:8200" 
+              cluster_addr = "https://edgex-vault:8201" 
+          } 
+          default_lease_ttl = "168h" 
+          max_lease_ttl = "720h"
+      - VAULT_CONFIG_DIR=/vault/config
+      - VAULT_UI=true
+    volumes:
+      - vault-file:/vault/file
+      - vault-logs:/vault/logs
+      - /run/edgex/secrets/edgex-vault:/run/edgex/secrets/edgex-vault:ro
+    depends_on:
+      - security-secrets-setup
+      - volume
+      - consul
+
+  security-secrets-setup:
+    image: edgexfoundry/docker-edgex-vault:1.1.0-dev
+    container_name: edgex-secrets-setup
+    hostname: edgex-secrets-setup
+    tmpfs:
+      - /run
+    #command: "generate"
+    #command: "security-secrets-setup generate"
+    command: "cache"
+    #command: "import"
+    volumes:
+      - secrets-setup-cache:/etc/edgex/pki
+      - /run/edgex/secrets/edgex-vault:/run/edgex/secrets/edgex-vault
+    depends_on:
+      - volume
+
+  vault-worker:
+    image: edgexfoundry/docker-edgex-vault-worker-go:1.0.0
+    container_name: edgex-vault-worker
+    hostname: edgex-vault-worker
+    command: ["--init=true", "--debug=false", "--wait=10", "--insureskipverify=false"]
```

Comment:
This line needs to be command: ["--init=true", "--debug=false", "--vault-interval=10", "--insecurekipverify=false"] .
We updated the vault-worker but not this compose file, so hope this can get in with the PR as well.

Reply:
--insecureskipverify=false